etc/nginx/nginx.conf.tmpl

```diff
@@ -8,7 +8,7 @@ pcre_jit on;
 pid /run/nginx.pid;
 
 events {
-    worker_connections {{ getenv "NGINX_WORKER_CONNECTIONS" "1000" }};
+    worker_connections {{ getenv "NGINX_WORKER_CONNECTIONS" "10000" }};
```

Comment:
A high default removes nginx itself as a failure point when there are many open connections, like may be the case if many clients tried to download many assets at once.